app/views/welcome/index.html.erb

```diff
@@ -5,6 +5,9 @@
 
 <%= link_to 'View Topics', topics_url %>
 </br>
+<%= link_to 'View Collections', collections_url %>
+</br>
```

Comment:
🤔 We're not focussed on styling yet, so I guess line breaks are cool for now (and we're already using them)